src/cascadia/TerminalApp/AppActionHandlers.cpp

```diff
@@ -166,6 +166,34 @@ namespace winrt::TerminalApp::implementation
         }
     }
 
+    void TerminalPage::_HandleCloseOtherPanes(const IInspectable& /*sender*/,
+                                              const ActionEventArgs& args)
+    {
+        if (const auto terminalTab{ _GetFocusedTabImpl() })
+        {
+            const auto activePane = terminalTab->GetActivePane();
+            if (terminalTab->GetRootPane() != activePane)
+            {
+                _UnZoomIfNeeded();
+
+                // Accumulate list of all unfocused leaf panes
+                std::deque<uint32_t> unfocusedPaneIds{};
+                terminalTab->GetRootPane()->WalkTree([&](auto p) {

```

Comment:
```suggestion
                std::vector<uint32_t> unfocusedPaneIds;
                terminalTab->GetRootPane()->WalkTree([&](auto&& p) {
```

If you then replace `std::greater` with `std::less` and `front`/`pop_front` with `back`/`pop_back` you can use a more efficient container.

Reply:
Thank you for this suggestion. I changed it in f405ce0